src/wazuh.py

```diff
@@ -61,12 +61,13 @@ def _update_filebeat_configuration(container: ops.Container, ip_ports: list[str]
     container.push(FILEBEAT_CONF_PATH, yaml.safe_dump(filebeat_config_yaml), encoding="utf-8")
 
 
+# Won't sacrify cohesion and readability to make pylint happier
```

Comment:
I'm wondering if moving `node_name` and `node_type` into the `if` statement making them branch scope (they're only used in that branch) won't remove the linter warning. I'm not sure how pylint consider local variables

Reply:
No, that doesn't do the trick

Reply:
Let's keep it as it is for now